src/EditorFeatures/CSharpTest/SplitStringLiteral/SplitStringLiteralCommandHandlerTests.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.Commands;
+using Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Text;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.SplitStringLiteral
+{
+    public class SplitStringLiteralCommandHandlerTests
+    {
+        private async Task TestWorkerAsync(
+            string inputMarkup, string expectedOutputMarkup, bool replaceSingleQuoteWithDouble, Action callback)
+        {
+            if (replaceSingleQuoteWithDouble)
+            {
+                inputMarkup = inputMarkup.Replace('\'', '"');
```

Comment:
Ugh! Seems like pretty surprising behavior in the tests just to avoid writing double quotes twice.